Makefile

```diff
@@ -88,3 +90,8 @@ check:
 		echo "The following differences will fail CI until committed:"; \
 		git diff --exit-code; \
 	fi
+
+.PHONY: site
+site: ## Serve website content
+	@git submodule update
```

Comment:
Does it work to add `--init` here? Or otherwise the user will have to run that manually once?

Reply:
will add sure